### PR TITLE
feat(mirinae): add boolean type to jsonSchemaForm

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA
image

### CODE REVIEW

File: packages/mirinae/src/inputs/forms/json-schema-form/PJsonSchemaForm.vue [hunk 1]
```diff
@@ -328,6 +337,7 @@ export default defineComponent<JsonSchemaFormProps>({
         /* Event Handlers */
         // form input case
         const handleUpdateFormValue = (property: InnerJsonSchema, propertyIdx: number, val?: any) => {
+            console.log('handleUpdateFormValue', property, propertyIdx, val);
```
Comment (yuda110) [inline packages/mirinae/src/inputs/forms/json-schema-form/PJsonSchemaForm.vue:340]: nitpick: console.log here!

**Review by yuda110 — APPROVED** [review]
LGTM~

**Review by piggggggggy — APPROVED** [review]
LGTM~

**Comment by sulmoJ** [comment]
ありがとうございます。

**Pull request merged**